script/test-cms-export.js

```diff
@@ -12,19 +14,54 @@ const {
   readEntity,
 } = require('../src/site/stages/build/process-cms-exports/helpers');
 
-const fileNames = readAllNodeNames();
-const entities = map(fileNames, entityDetails =>
-  readEntity(contentDir, ...entityDetails),
+const optionDefinitions = [
+  { name: 'count', alias: 'c', type: Number },
+  { name: 'node', alias: 'n', type: String },
+  { name: 'print', alias: 'p', type: Number },
+];
+
+const { count, node: nodeName, print: printIndex } = commandLineArgs(
+  optionDefinitions,
 );
 
-// Assemble all the nodes
-// const modifiedEntities = map(entities, entity => assembleEntityTree(entity));
+if (nodeName) {
+  const nodeNamePieces = nodeName.split('.').slice(0, 2);
+  assert(
+    nodeNamePieces.length === 2,
+    'Node needs to be the filename of an entity. E.g. node.<uuid>.json',
+  );
 
-// Assemble only the first node for debugging
-const modifiedEntities = map([entities[0]], entity =>
-  assembleEntityTree(entity),
-);
+  const node = assembleEntityTree(readEntity(contentDir, ...nodeNamePieces));
+  // eslint-disable-next-line no-console
+  console.log(JSON.stringify(node, null, 2));
+} else {
+  // Make sure the printIndex is valid if passed
+  if (printIndex) {
+    if (count)
+      assert(
+        printIndex < count,
+        `Print index (${printIndex}) must be less than node count (${count}) to point to a valid node.`,
+      );
+    assert(printIndex >= 0, `Print index (${printIndex}) must be >= 0`);
+  }
+
+  const fileNames = readAllNodeNames();
+  const entities = map(fileNames, entityDetails =>
+    readEntity(contentDir, ...entityDetails),
+  );
+
+  const modifiedEntities = map(
+    entities.slice(0, count || entities.length),
+    entity => assembleEntityTree(entity),
+  );
 
-// eslint-disable-next-line
-console.log('Number of files:', modifiedEntities.length);
-// console.log('First node:', JSON.stringify(modifiedEntities[0], null, 2));
+  // eslint-disable-next-line no-console
+  console.log('Number of files:', modifiedEntities.length);
+  if (printIndex) {
+    // eslint-disable-next-line no-console
+    console.log(
+      `Node ${printIndex}:`,
+      JSON.stringify(modifiedEntities[printIndex || 0], null, 2),
```

Comment:
Related to the comment above - I think we could ensure that `printIndex` will have a value.

Reply:
Works for me. :+1:

Reply:
Actually, after poking at it for a minute more, I realized the way I had it meant we couldn't tell it to _not_ print a record if we set the default value. I changed the `if` statement above this to check if it's not `undefined` (this way we can actually print the first record `0`).